src/common/util/include/openvino/util/file_util.hpp

```diff
@@ -125,6 +125,15 @@ bool is_absolute_file_path(const std::string& path);
  */
 void create_directory_recursive(const std::string& path);
 
+#ifdef OPENVINO_ENABLE_UNICODE_PATH_SUPPORT
+/**
+ * @brief Interface function to create directorty recursively by given path
+ * @param path - path to file wide-string, can be relative to current working directory
+ * @throw runtime_error if any error occurred
+ */
+void create_directory_recursive(const std::wstring& path);
```

Comment:
```suggestion
void create_directory_recursive_w(const std::wstring& path);
```

Reply:
why here need the different name for wstring? The other api also using the same name overload